src/test/cases/coercion_not_defined/ancestry.expectations

```diff
@@ -0,0 +1,15 @@
+{
+  "summary": [""],
```

Comment:
This is weird. Looks like the `""` in all this ancestry represents the nameless default namespace.

Maybe we could tidy this later... but since this wasn't a change caused by this PR I'll let it slide for now.